frontend/src/features/admin-form/AdminFormResponsesPage.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { MemoryRouter, Route } from 'react-router'
+import { Meta, Story } from '@storybook/react'
+
+import { getAdminFormResponse } from '~/mocks/msw/handlers/admin-form'
+
+import { viewports } from '~utils/storybook'
+
+import { AdminFormPage } from './common/AdminFormPage'
+
+export default {
+  title: 'Pages/AdminFormPage/Responses',
+  component: AdminFormPage,
+  decorators: [
+    (storyFn) => {
+      return (
+        <MemoryRouter initialEntries={['/admin/form/1234/responses']}>
```

Comment:
why do we need `MemoryRouter` for these? is it to keep track of changes in the tabs without changing the URL?

Reply:
It's so `react-router-dom#Link` components can work (and also to control which tab the page initially renders). Will add a comment.